pkgs/by-name/ec/ecapture/package.nix

```
@@ -0,0 +1,126 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  stdenv,
+  bpftool,
```

```suggestion
  bpftools,
```

`bpftool` is a deprecated alias of `bpftools`.